homeassistant/components/notify/facebook.py

```diff
@@ -55,20 +58,28 @@ def send_message(self, message="", **kwargs):
             _LOGGER.error("At least 1 target is required")
             return
 
-        for target in targets:
-            # If the target starts with a "+", we suppose it's a phone number,
-            # otherwise it's a user id.
-            if target.startswith('+'):
-                recipient = {"phone_number": target}
-            else:
-                recipient = {"id": target}
-
-            body = {
-                "recipient": recipient,
-                "message": body_message
+        # broadcast message
+        if targets[0].startswith('BROADCAST'):
+
+            brdcast_create_body = {"messages": [body_message]}
```

Comment:
Please use meaningful variable names. This doesn't look like an abbreviation but more as a word with typos.